courses/free-courses-hi.md

```diff
@@ -181,6 +181,7 @@
 * [All C Concepts \| Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
 * [C Language for GATE — Complete Playlist](https://www.youtube.com/playlist?list=PLC36xJgs4dxG-IqARhc23jYTDMYt7yvZP) - Gate CSE lectures by Amit Khurana
 * [C Language Tutorial for Beginners (with Notes & Practice Questions)](https://www.youtube.com/watch?v=irqbmMNs2Bo) - Apna College
+* [C Language Tutorial for Beginners (With Notes + Surprise) 🔥](https://youtu.be/aZb0iu4uGwA?si=WX7d5lobVBpjgwbH) - CodeWithHarry
```

Comment:
no emoji. no shortened links. Maybe remove the old one?
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Actually the emoji was there in the original title, but anyways, I've removed it. I've also removed the old one. If these changes seem okay, please merge the pull request.